File: Data Structures & Algorithms/anagram-groups/submission-0.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
    map <string, vector<string>> mp;
    for (string x : strs){
        string tmp = x;
        sort(tmp.begin(), tmp.end());
        mp[tmp].push_back(x);
        }
    vector<vector<string>> res;
    for (auto it : mp){
        res.push_back(it.second);   
    }
    return res;
    }
};

File: Data Structures & Algorithms/valid-palindrome-ii/submission-0.py
class Solution:
    def validPalindrome(self, s: str) -> bool:
        if s[::-1] == s:
            return True
        for i in range(len(s)):
            tmp = s[0:i] + s[i+1:]
            if tmp[::-1] == tmp:
                return True
        return False
            